readme.md

```diff
@@ -107,6 +107,7 @@ Similarly to Tobira, Minna no Nihongo is intended for intermediate-level student
 - [LingoDeer](https://www.lingodeer.com/) - Japanese course on a new learning app similar to Duolingo, but made with a focus on East Asian languages.:iphone:
 - [NativShark](https://nativshark.com/) - All in one Japanese learning platform, combining vocab, grammar, kanji and speaking practice.:moneybag:
 - [NHK 高校講座 / NHK High School Course](https://www.nhk.or.jp/kokokoza/) - Video or audio of high school course made by NHK.Easy to study the basic glossaries of different field.:man:
+- [Drops: Language learning app](https://apps.apple.com/it/app/drops-language-learning-app/id939540371) - Duolingo-like app which offers 5 minutes/day free flashcards-based app separated by categories. :iphone:
```

Comment:
drops official website: https://languagedrops.com/ offer links to both android and ios